src/ic-certified-assets/src/lib.rs

```diff
@@ -693,6 +693,23 @@ fn http_request(req: HttpRequest) -> HttpResponse {
                 encodings.push(v.trim().to_string());
             }
         }
+        if name.eq_ignore_ascii_case("Host") {
+            if let Some(host) = value.split(':').next() {
+                if host.contains("raw.ic0.app") {
+                    let replacement_url = if host.contains("http") {
```

Comment:
What is the expected behavior if the host value is `my.http.files.raw.ic0.app` ?

Reply:
I can probably remove that check, I wasn't sure if the host header could contain the URL scheme but it looks like it can't.